docs/WindowsBuild.md

```diff
@@ -152,8 +160,13 @@ cmake -G "Ninja" "%swift_source_dir%/swift"^
  -DSWIFT_INCLUDE_TESTS=FALSE^
  -DCMAKE_C_COMPILER="%llvm_bin_dir%/clang-cl.exe"^
  -DCMAKE_CXX_COMPILER="%llvm_bin_dir%/clang-cl.exe"^
- -DCMAKE_C_FLAGS="-fms-compatibility-version=19.00 /Z7"^
- -DCMAKE_CXX_FLAGS="-fms-compatibility-version=19.00 -Z7" ^
+ -DCMAKE_C_FLAGS="-fms-compatibility-version=19.00 /Z7 -Wno-c11-extensions -Wno-language-extension-token -Wno-reserved-id-macro -Wno-disabled-macro-expansion"^
```

Comment:
I don't think that the flags are needed any longer.  The warnings that are superfluous we should be disabling a different way I think.

Reply:
We still get these warnings with the TOT code:

```
C:\Users\dplan\Documents\Developer\swift-repos\swift\stdlib\public\runtime\ImageInspectionCOFF.cpp(106,9):  warning: 'auto' type specifier is incompatible with C++98 [-Wc++98-compat]
  const auto &protocols_section = sections->swift5_protocols;
        ^~~~

C:\Users\dplan\Documents\Developer\swift-repos\swift\stdlib\public\runtime\SwiftDtoa.cpp(319,12):  warning: use of old-style cast [-Wold-style-cast]
        = ((uint64_t)1 << significandBitCount) - 1;
           ^         ~
```

How else would you turn those expected warnings off? I'm not too concerned with whether this part looks good or not. My main concern is to increase the chance that folks who are interested in working on the Windows port will see the problematic warnings (especially the ones about shortening 64bit values to 32bit values without an explicit cast as a conformation that, yes, someone checked that this to ensure that this is actually correct code). 

Eventually we'll want to use the build script on Windows anyway and there shouldn't be a need anymore to document the low-level build steps.

Reply:
I think that a couple of these options are fine, but I would rather that we do it through CMake so that they work even when cross-compiling.